addons/website/tests/template_qweb_test.xml

```diff
@@ -2,15 +2,15 @@
 <odoo>
     <template id="website.test_bundle">
         <script type="text/javascript" src="/web/static/lib/qweb/qweb2.js"></script>
-        <script type="text/javascript" src="http://test.external.link/javascript1.js"></script>
+        <script type="text/javascript" src="http://www.example.com/javascript1.js"></script>
```

Comment:
Does linking to example.com actually satisfy debian? I know that these are reserved by IANA but they're still an actual domain actually served by something.

Reply:
I hope so. I'll just ask the question but seems safer than an non existing domain.

Reply:
The bug report does *not* clearly say that these links have to be changed. Sebastien Delafond only says "Some of these are harmless placeholders used during testing, but others need to be clarified and fixed with upstream's help."

This requires a clarification of the "rule" or basis for the change, otherwise it will be broken again in future commits.

Reply:
FWIW, [the lintian check](https://github.com/Debian/lintian/blob/13ccfaa7571f5478157a4bd92e1bfc9754ab7bea/checks/files.pm#L1988-L1990) seems to only special-case `localhost`, so that would be the only option?

Also, what does ["Certainty: wild-guess"](https://github.com/Debian/lintian/blob/master/checks/files.desc#L1023) actually mean? At least that's a good assessment of the reliability of this test...